### PR TITLE
Firewall: Update images alt text

### PR DESCRIPTION
**Pull request by pedrosousa**
Partially addresses https://jira.cfops.it/browse/PCX-3394.

### CODE REVIEW

File: content/firewall/cf-dashboard/create-edit-delete-rules.md [hunk 1]
```diff
@@ -52,60 +46,52 @@ To create a new firewall rule:
...
-
 In the confirmation dialog that appears, click **Delete** to confirm and complete the operation.
 
 ### Order rules
 
-By default, Cloudflare evaluates firewall rules in **list order**, where rules are evaluated in the order they appear in the **Rules List**. When list ordering is enabled, the Rules List allows you to drag and drop firewall rules into position, as shown below.
+By default, Cloudflare evaluates firewall rules in **list order**, where rules are evaluated in the order they appear in the rules list. When list ordering is enabled, the rules list allows you to drag and drop firewall rules into position, as shown below.
 
-![Order firewall rules](/firewall/static/firewall-rules-expression-builder-10.gif)
+![Animation of a user dragging and dropping a rule in the rules list to reorder it](/firewall/static/firewall-rules-expression-builder-10.gif)
```
Comment (deadlypants1973) [inline content/firewall/cf-dashboard/create-edit-delete-rules.md:89]: I believe according to alt text guidelines, you can drop the 'animation of' part here.

File: content/firewall/cf-dashboard/rules-lists/_index.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ IP Lists are stored at the account level. To access the **Lists** interface:
 
 The **Lists** user interface displays.
 
-![Lists UI](/firewall/static/lists-ui.png)
+![The Lists user interface in the Cloudflare dashboard, where you can create and manage IP Lists](/firewall/static/lists-ui.png)
```
Comment (deadlypants1973) [inline content/firewall/cf-dashboard/rules-lists/_index.md]:
```suggestion
![The Lists user interface in the Cloudflare dashboard where you can create and manage IP Lists](/firewall/static/lists-ui.png)
```

File: content/firewall/cf-firewall-rules/order-priority.md [hunk 3]
```diff
@@ -20,47 +20,48 @@ Cloudflare Firewall Rules is part of a larger evaluation chain for HTTP requests
...
 
-![The user drags a firewall rule in the Rules List to reorder it](/firewall/static/firewall-rules-order-and-priority-2.gif)
+![Animation of a firewall rule being moved into a new position in the rules list to reorder it](/firewall/static/firewall-rules-order-and-priority-2.gif)
 
 Once there are more than 200 total rules, including inactive rules, you must manage evaluation using priority ordering. When you cross this threshold, the firewall rules interface automatically switches to priority ordering.
 
 ## Managing rule evaluation by priority order
 
-Although priority ordering is enabled automatically when the number of active and inactive firewall rules exceeds 200, you can manually enable priority ordering at any time from the **Rules List**. To manually enable priority ordering, click **Ordering** and then select the **Priority Numbers** radio button, as shown below:
+Although priority ordering is enabled automatically when the number of active and inactive firewall rules exceeds 200, you can manually enable priority ordering at any time from the rules list.
```
Comment (deadlypants1973) [inline content/firewall/cf-firewall-rules/order-priority.md]:
```suggestion
Although priority ordering is enabled automatically when the number of active and inactive firewall rules exceeds 200, you can manually enable priority ordering at any time from the rules list.
```

**Review by deadlypants1973 — APPROVED** [review]

File: content/firewall/cf-dashboard/create-edit-delete-rules.md (hunk 1 above)
Comment (pedrosousa) [reply content/firewall/cf-dashboard/create-edit-delete-rules.md:89]:
```suggestion
![A user dragging and dropping a rule in the rules list to reorder it](/firewall/static/firewall-rules-expression-builder-10.gif)
```

File: content/firewall/cf-dashboard/create-edit-delete-rules.md [hunk 4]
```diff
@@ -52,60 +46,52 @@ To create a new firewall rule:
...
-
 In the confirmation dialog that appears, click **Delete** to confirm and complete the operation.
 
 ### Order rules
 
-By default, Cloudflare evaluates firewall rules in **list order**, where rules are evaluated in the order they appear in the **Rules List**. When list ordering is enabled, the Rules List allows you to drag and drop firewall rules into position, as shown below.
+By default, Cloudflare evaluates firewall rules in **list order**, where rules are evaluated in the order they appear in the rules list. When list ordering is enabled, the rules list allows you to drag and drop firewall rules into position, as shown below.
 
-![Order firewall rules](/firewall/static/firewall-rules-expression-builder-10.gif)
+![A user dragging and dropping a rule in the rules list to reorder it](/firewall/static/firewall-rules-expression-builder-10.gif)
```
Comment (pedrosousa) [inline content/firewall/cf-dashboard/create-edit-delete-rules.md]:
```suggestion
![Animation of a user dragging and dropping a rule in the rules list to reorder it](/firewall/static/firewall-rules-expression-builder-10.gif)
```
Adding back "Animation of", after a quick internal discussion.

**Pull request merged**